api/base/utils.py

```diff
@@ -23,11 +24,9 @@
 from website import settings as website_settings
 from website import util as website_util  # noqa
 
-# These values are copied from rest_framework.fields.BooleanField
-# BooleanField cannot be imported here without raising an
-# ImproperlyConfigured error
-TRUTHY = set(('t', 'T', 'true', 'True', 'TRUE', '1', 1, True, 'on', 'ON', 'On', 'y', 'Y', 'YES', 'yes'))
-FALSY = set(('f', 'F', 'false', 'False', 'FALSE', '0', 0, 0.0, False, 'off', 'OFF', 'Off', 'n', 'N', 'NO', 'no'))
+# See https://github.com/encode/django-rest-framework/blob/3.13.1/rest_framework/fields.py#L699-L721
+TRUTHY = fields.BooleanField.TRUE_VALUES
+FALSY = fields.BooleanField.FALSE_VALUES
```

Comment:
* `ImproperlyConfigured` gone, no longer a blocker
* The only difference is that [3.13.1](https://github.com/encode/django-rest-framework/blob/3.13.1/rest_framework/fields.py#L699-L721) has an extra `'Yes'` and `'No'` added to respective set.